frontend/app/package.json

```diff
@@ -57,6 +57,10 @@
     ]
   },
   "dependencies": {
+    "@emotion/react": "^11.10.4",
```

Comment:
Are the version of these dependencies updated? Agus made some fixes here to have all the environment working